src/plugins/unified_search/public/filter_bar/filter_editor/filter_editor.tsx

```diff
@@ -170,12 +170,14 @@ class FilterEditorUI extends Component<FilterEditorProps, State> {
 
           <EuiPopoverFooter paddingSize="s">
             <EuiFlexGroup direction="rowReverse" alignItems="center" responsive={false}>
```

Comment:
We can also fix this by creating a new stacking context using [isolation](https://developer.mozilla.org/en-US/docs/Web/CSS/isolation) on button group container:

```suggestion
            <EuiFlexGroup style={{ isolation: 'isolate' }} direction="rowReverse" alignItems="center" responsive={false}>
```

Reply:
Sure I can go with this solution too.